### PR TITLE
4th hw (first commit)

### PR DESCRIPTION
**Pull request by Daria04**

### CODE REVIEW

File: src/test/java/com/epam/tc/hw4/test/FirstExerciseTestFailure.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    @AfterMethod
+    public void takeScreenshotOnFailure(ITestResult result) throws IOException {
+        if (result.getStatus() == ITestResult.FAILURE) {
+            System.out.println(result.getStatus());
+            File screenFile = ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);
+            FileUtils.copyFile(screenFile, new File("errorScreenshot\\"
+                    + result.getName() + Arrays.toString(result.getParameters()) + ".jpg"));
+
+        }
+    }
```
Comment (DmitryKhodakovsky) [inline src/test/java/com/epam/tc/hw4/test/FirstExerciseTestFailure.java]: Why do you need this code here? It should be placed in `ScreenshotListener` class

**Review by DmitryKhodakovsky — CHANGES_REQUESTED** [review]
Please review comment

File: src/test/java/com/epam/tc/hw4/test/FirstExerciseTestFailure.java (hunk 1 above)
Comment (Daria04) [reply src/test/java/com/epam/tc/hw4/test/FirstExerciseTestFailure.java]: I did not know exactly where to put this method. Now it is located in the right place.

**Review by DmitryKhodakovsky — APPROVED** [review]
Approved

**Pull request merged**